doc/lua_api.txt

```diff
@@ -5832,15 +5832,88 @@ This is basically a reference to a C++ `ServerActiveObject`
+              skybox during the night. (default: `#006aff`)
+            * `night_horizon`: ColorSpec, for the bottom half of the `"regular"`
+              skybox during the night. (default: `#4090ff`)
+            * `indoors`: ColorSpec, for when you're either indoors or 
+              underground. Only applies to the `"regular"` skybox.
+              (default: `#646464`)
+            * `sun_tint`: ColorSpec, changes the fog tinting for the sun
+              at sunrise and sunset.
+            * `moon_tint`: ColorSpec, changes the fog tinting for the moon
+              at sunrise and sunset.
+            * `tint_type`: ColorSpec, changes which mode the directional fog
+                abides by, `"custom"` uses `sun_tint` and `moon_tint`, while
+                `"default"` uses the classic Minetest sun and moon tinting.
+                Will use tonemaps, if set to `"default"`. (default: `"default"`)
+* `get_sky()`: returns a table with the current sky parameters as in
+    `set_sky`.
+* `set_sun(parameters)`:
+    * `parameters` is a table with the following optional fields:
+        * `visible`: Boolean for whether the sun is visible.
+            (default: `true`)
+        * `texture`: A regular texture for the sun. Setting to `""`
+            will re-enable the mesh sun. (default: `"sun.png"`)
+        * `tonemap`: A 512x1 texture containing the tonemap for the sun
+            (default: `"sun_tonemap.png"`)
+        * `sunrise`: A regular texture for the sunrise texture.
+            (default: `"sunrisebg.png"`)
+        * `sunrise_visible`: Boolean for whether the sunrise texture is visible.
+            (default: `true`)
+        * `rotation`: Floating point value to set what direction the
+            sun should rotate around the world. (default: `90`)
+        * `scale`: Float controlling the overall size of the sun. (default: `1`)
+* `get_sun()`: returns a table with the current sun parameters as in
+    `set_sun`.
+* `set_moon(parameters)`:
+    * `parameters` is a table with the following optional fields:
+        * `visible`: Boolean for whether the moon is visible.
+            (default: `true`)
+        * `texture`: A regular texture for the moon. Setting to `""`
+            will re-enable the mesh moon. (default: `"moon.png"`)
+        * `tonemap`: A 512x1 texture containing the tonemap for the moon
+            (default: `"moon_tonemap.png"`)
+        * `rotation`: Floating point value to set what direction the
+            moon should rotate around the world. (default: `-90`)
+        * `scale`: Float controlling the overall size of the moon (default: `1`)
+* `get_moon()`: returns a table with the current moon parameters as in
+    `set_moon`.
+* `set_stars(parameters)`:
+    * `parameters` is a table with the following optional fields:
+        * `visible`: Boolean for whether the stars are visible.
+            (default: `true`)
+        * `count`: Interger number to set the number of stars in 
```

Comment:
'Integer'

Reply:
Not resolved. Why did you resolve this? Did you resolve it before making changes then forget?

Reply:
Oops, I was checking them off as I made the changes locally.

Reply:
Now fixed.